packages/core/src/modules/ledger/LedgerApi.ts

```diff
@@ -75,14 +84,33 @@ export class LedgerApi {
 
     const schemaId = generateSchemaId(did, schema.name, schema.version)
 
+    // Generate teh qualified ID
+    // FIXME: How to handle indyLedgers[0]? This should not require selecting the first on eby default
```

Comment:
We should get the ledgerWritePool from the ledgerservice

Reply:
ahhhh alright. Thx!